src/swagger/api.yaml

```diff
@@ -26122,27 +26122,25 @@ components:
         - DELTA_CRL
         - OCSP
     IdentityProviderPolicy:
-      allOf:
-        - $ref: '#/components/schemas/Policy'
-        - type: object
-          properties:
-            accountLink:
-              $ref: '#/components/schemas/PolicyAccountLink'
-            conditions:
-              $ref: '#/components/schemas/PolicyRuleConditions'
-            mapAMRClaims:
-              type: boolean
-              description: Enable mapping AMR from IdP to Okta to downstream apps
-              default: false
-              x-okta-lifecycle:
-                features:
-                  - IDP_AMR_CLAIMS_MAPPING
-            maxClockSkew:
-              type: integer
-            provisioning:
-              $ref: '#/components/schemas/Provisioning'
-            subject:
-              $ref: '#/components/schemas/PolicySubject'
+      type: object
+      properties:
```

Comment:
IdentityProviderPolicy definition - ref: https://developer.okta.com/docs/reference/api/idps/#policy-object

Reply:
You have to manually do this?